e2e/testcafe-devextreme/tests/chat/typingIndicator.ts

```diff
@@ -14,13 +14,29 @@ fixture.disablePageReloads`ChatTypingIndicator`
 test('Chat: typing indicator with emptyview', async (t) => {
   const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
 
-  await testScreenshot(t, takeScreenshot, 'Typing indicator with emptyview.png', { element: '#chat' });
+  const chat = new Chat('#container');
+
+  await testScreenshot(t, takeScreenshot, 'Typing indicator with emptyview.png', {
+    element: '#container',
+    shouldTestInCompact: true,
+    compactCallBack: async () => {
+      await chat.repaint();
+    },
+  });
+
+  const darkTheme = getFullThemeName().replace('light', 'dark');
+  await testScreenshot(t, takeScreenshot, 'Typing indicator with emptyview.png', {
+    element: '#container',
+    theme: darkTheme,
+    themeChanged: async () => {
+      await chat.repaint();
+    },
+  });
```

Comment:
I see u added a screenshot taking for compact and dark + removed previous `testScheenshot`.
Do we test non-compact light now?